signer/src/testing/wsts.rs

```diff
@@ -409,6 +449,27 @@ impl SignerSet {
         )
     }
 
+    /// Participate in N signing rounds
+    pub async fn participate_in_signing_rounds_forever(&mut self) {
+        loop {
+            self.participate_in_signing_round().await
+        }
+    }
```

Comment:
Why forever and not N signing rounds?

Reply:
Oh the comment is stale. I first implemented it for N signing rounds, only to find that the number of transactions constructed varied between rounds (which is strange, but our tests are unfortunately not completely deterministic).

I did not find a good way to control this number in the test case, so I just wrote the test case to not assume a fixed number instead.

Reply:
Updated the comment.